src/main/java/core/basesyntax/db/FileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    public FileReaderImpl(String s) {

```

Comment:
The constructor FileReaderImpl(String s) is not used and does not perform any operation. Consider removing it if it's not needed.

Reply:
ok, done